gh-actions/common/build-debian/action.yml

```diff
@@ -12,14 +12,23 @@ inputs:
   token:
     required: false
     description: If provided, used for git authentication in the source build
+  extra-source-build-deps:
+    description: A list of extra build dependencies required during source build.
+    required: false
+    # FIXME: this should default to '', but we don't want to break job depending on us for now
+    default: 'ca-certificates git'
+  DEB_BUILD_OPTIONS:
```

Comment:
I have removed `DEB_BUILD_OPTIONS`, but IMHO it can be quite convenient for testing `nocheck` builds too, which are somewhat supported by the ecosystem (other than useful to do simpler test-builds).

Regarding `extra-source-build-deps` I think we still need it as explained in the commit, and the FIXME, IMHO is very wrong always installing `ca-certificates` and `git` even on source builds.

So, if you want I can make this more strict by not allowing people to customize what install (I need a good variable name though :)), but my point here is that all the projects that really need this should define it, not this low-level action.

Reply:
I think our goal is really to add options when we have a first consumer requesting for it and revisting the rationale.

Reply:
btw, my comment was (see my first comment) for both options.